modules/bitcoind.nix

```diff
@@ -9,6 +9,9 @@ let
     ${optionalString cfg.testnet "testnet=1"}
     ${optionalString (cfg.dbCache != null) "dbcache=${toString cfg.dbCache}"}
     ${optionalString (cfg.prune != null) "prune=${toString cfg.prune}"}
+    sysperms=${if cfg.sysperms then "1" else "0"}
+    disablewallet=${if cfg.disablewallet then "1" else "0"}
```

Comment:
You can also have both options optional by declaring their type as `types.nullOr types.bool` and then using `${optionalString (cfg.sysperms != null) "sysperms=${if cfg.sysperms then "1" else "0"}"}`.

Reply:
Fixed in fca4af5